### ISSUE TITLE
 FlowManager.getDatabase(Db.class).reset() not recreate database

### ISSUE THREAD

**Issue by PingForward** [Raizlabs/DBFlow#1089]
DBFlow Version: 4.0.0-beta2
Bug

Description: 
` FlowManager.getDatabase(AppDatabase.class).reset(getApplicationContext());`

**I expected to reset my database (delete all data, and reopen it for new transactions).** 
So all data is deleted when I call this method but database file not created again.
I tried to call 
```
FlowManager.init(new FlowConfig.Builder(context)
                .openDatabasesOnInit(true).build());
```
but no effect.
And if a new transaction happen I got this message:
`java.lang.IllegalStateException: Cannot perform this operation because the connection pool has been closed.`

any help?

**Comment by trevjonez** [comment]
duplicate of https://github.com/Raizlabs/DBFlow/issues/925 ?

**Comment by agrosner** [comment]
in `develop` fix for #925

**Comment by CapnSpellcheck** [comment]
Just to be clear. After executing `FlowManager.getDatabase("Main").reset(this)` we should execute whatever we did to initialize the database in the Application class?

```
        val fc = FlowConfig.Builder(this)
            .addDatabaseConfig(DatabaseConfig.Builder(Database::class.java).build())
            .build()
        FlowManager.init(fc)
```
